application/java/test/org/openda/costa/CostaOpendaAppTest.java

```diff
@@ -46,29 +45,29 @@ public static void testDummy() {
 		//            the test below is de-activated by renaming it to tst...()
 	}
 
-	public void testOscill_EnKF() throws IOException {
+	public void testOscill_EnKF() {
         ApplicationRunner.setRunningInTest(true);
         File config = new File(testRunDataDir, "OscillEnKFOpenDaConfig.xml");
-        String args[] = new String[1];
+        String[] args = new String[1];
         args[0] = config.getAbsolutePath();
         org.openda.application.OpenDaApplication.main(args);
         System.out.println("testOscill_EnKF DONE");
     }
 
-    public void tstOscill_Simplex() throws IOException {
+    public void tstOscill_Simplex() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The instance method name 'tstOscill_Simplex' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=5102854)